sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/IntervalExpressionsSuite.scala

```diff
@@ -661,4 +661,17 @@ class IntervalExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
         }
     }
   }
+
+  test("SPARK-36431: Support comparison of ANSI intervals with different fields") {
```

Comment:
The UT here should not test type coercion. Let's remove it.

Reply:
Done